src/runtime/meter-toast-api.js

```diff
@@ -29,6 +29,8 @@ export const IFRAME_BOX_SHADOW =
   'rgba(60, 64, 67, 0.3) 0px -2px 5px, rgba(60, 64, 67, 0.15) 0px -5px 5px';
 export const MINIMIZED_IFRAME_SIZE = '420px';
 
+const AUTO_PINGBACK_TIMEOUT = 5000;
```

Comment:
I would prefer this be 10-15 seconds. 5 seconds seems like a somewhat normal loading time on slow connections.

IMO this should be the longest we should expect our users to wait before we bail on the toast request.

Reply:
Also add a comment please :)